docs/pages/trustedclusters.mdx

```diff
@@ -308,7 +308,7 @@ spec:
   enabled: true
   role_map:
     - remote: admin
```

Comment:
We've moved away from the Admin Role. I would make this 
```suggestion
    - remote: access
```

Reply:
Will make this change - the Trusted Clusters docs are modified here: https://github.com/gravitational/teleport/pull/6999/files

Reply:
I will also make the change here. Thanks!